### PR TITLE
DOCS-1267: Refactor Installation Guide

### PR DESCRIPTION
**Pull request by npentrel**
Addressing suggestions from the UX research. I am not sure how we can make the next steps card less tangential. If you have an idea, please feel free to assign the rest of the ticket back to yourself once this merges

### CODE REVIEW

File: docs/installation/_index.md [hunk 1]
```diff
@@ -73,17 +73,22 @@ If you have a [compatible operating system](/installation/), follow along with t
...
+
+1. Go to the [Viam app](https://app.viam.com) and add a new robot by providing a name in the **New Robot** field and clicking **Add robot**.
    If this is your first time using the Viam app, you must create an account first.
 
+   ![The 'First Location' page on the Viam app with a new robot name in the New Robot field and the Add robot button next to the field highlighted.](/manage/app-usage/create-robot.png)
+
 1. On the **Setup** tab, select `Mac` as the **Architecture**.
 
-1. Follow the steps shown on the **Setup** tab to install `viam-server` on your macOS computer.
+1. Follow the steps shown on the **Setup** tab to install `viam-server` on your Linux computer.
```
Comment (andf-viam) [inline docs/installation/_index.md:88]:
```suggestion
1. Follow the steps shown on the **Setup** tab to install `viam-server` on your macOS computer.
```

File: docs/installation/_index.md [hunk 2]
```diff
@@ -73,17 +73,22 @@ If you have a [compatible operating system](/installation/), follow along with t
...
 
-1. Once `viam-server` is installed and running, return to the **Setup** page on the [Viam app](https://app.viam.com) and wait for confirmation that your computer has successfully connected.
+1. Once you have followed the steps on the **Setup** tab `viam-server` is installed and running.
```
Comment (andf-viam) [inline docs/installation/_index.md:90]:
```suggestion
1. Once you have followed the steps on the **Setup** tab, `viam-server` is installed and running.
```

File: docs/installation/prepare/jetson-agx-orin-setup.md [hunk 3]
```diff
@@ -109,6 +109,14 @@ For a Jetson AGX Orin, reference the following:
 Note that I2C buses do not need to be configured through <file>jetson-io.py</file>.
 See NVIDIA's documentation on [Configuring the Jetson Expansion Headers](https://docs.nvidia.com/jetson/archives/r35.1/DeveloperGuide/text/HR/ConfiguringTheJetsonExpansionHeaders.html) for more information.
 
+## Next Steps
+
+{{< cards >}}
+{{% card link="/manage/configuration/" %}}
+{{% card link="/tutorials/" %}}
+{{% card link="/try-viam/" %}}
```
Comment (andf-viam) [inline docs/installation/prepare/jetson-agx-orin-setup.md:117]: With `viam-server` installed on my own hardware, much of the allure and immediate usefulness of Try is lost. How about Program (SDKs) instead?

```suggestion
{{% card link="/program/" %}}
```

recommend making this change on all the other prepare board procedurals too.

File: docs/installation/_index.md [hunk 4]
```diff
@@ -73,17 +73,22 @@ If you have a [compatible operating system](/installation/), follow along with t
...
+   Return to the **Setup** page on the Viam app and wait for confirmation that your computer has successfully connected.
```
Comment (andf-viam) [inline docs/installation/_index.md:91]: Out-of-diff recommendation, see comment below:

Line 102:

```
- {{% card link="/try-viam/" %}}
+ {{% card link="/program/" %}}
```

File: static/include/install/install-linux-aarch.md [hunk 5]
```diff
@@ -3,13 +3,16 @@ The AppImage is a single, self-contained binary that runs on 64-bit Linux system
...
    If this is your first time using the Viam app, you must create an account first.
 
+   ![The 'First Location' page on the Viam app with a new robot name in the New Robot field and the Add robot button next to the field highlighted.](/manage/app-usage/create-robot.png)
+
 1. On the **Setup** tab, select `Linux (Aarch64)`.
 
 1. Follow the steps shown on the **Setup** tab to install `viam-server` on your Linux computer.
 
-1. Once `viam-server` is installed and running, return to the **Setup** page on the [Viam app](https://app.viam.com) and wait for confirmation that your computer has successfully connected.
+1. Once you have followed the steps on the **Setup** tab `viam-server` is installed and running.
```
Comment (andf-viam) [inline static/include/install/install-linux-aarch.md:15]:
```suggestion
1. Once you have followed the steps on the **Setup** tab, `viam-server` is installed and running.
```

File: static/include/install/install-linux.md [hunk 6]
```diff
@@ -29,14 +29,17 @@ To install `viam-server` on a Linux computer:
...
 
+   ![The 'First Location' page on the Viam app with a new robot name in the New Robot field and the Add robot button next to the field highlighted.](/manage/app-usage/create-robot.png)
+
 1. On the **Setup** tab, select `Linux (Aarch64)` or `Linux (x86_64)` for the appropriate **Architecture** for your computer.
    On most Linux operating systems, you can run `uname -m` to confirm your computer's architecture.
 
 1. Follow the steps shown on the **Setup** tab to install `viam-server` on your Linux computer.
 
-2. Once `viam-server` is installed and running, return to the **Setup** page on the [Viam app](https://app.viam.com) and wait for confirmation that your computer has successfully connected.
+1. Once you have followed the steps on the **Setup** tab `viam-server` is installed and running.
```
Comment (andf-viam) [inline static/include/install/install-linux.md:42]:
```suggestion
1. Once you have followed the steps on the **Setup** tab, `viam-server` is installed and running.
```

**Review by andf-viam — APPROVED** [review]
This looks great! The inlined "add a robot" step, bumped up Next steps section, and unlinked external resources really help to streamline the flow from start to finish. I think this is a great improvement, thank you!

File: docs/installation/prepare/jetson-nano-setup.md [hunk 7]
```diff
@@ -97,6 +97,14 @@ For a Jetson Orin Nano, reference the following:
 Note that I2C buses do not need to be configured through <file>jetson-io.py</file>.
 See NVIDIA's documentation on [Configuring the Jetson Expansion Headers](https://docs.nvidia.com/jetson/archives/r35.1/DeveloperGuide/text/HR/ConfiguringTheJetsonExpansionHeaders.html) for more information.
 
+## Next Steps
+
+{{< cards >}}
+{{% card link="/manage/configuration/" %}}
+{{% card link="/tutorials/" %}}
+{{% card link="/try-viam/" %}}
```
Comment (npentrel) [inline docs/installation/prepare/jetson-nano-setup.md:105]:
```suggestion
{{% card link="/program/" %}}
```

File: docs/installation/prepare/microcontrollers/_index.md [hunk 8]
```diff
@@ -56,6 +56,8 @@ See [Micro-RDK](/micro-rdk/) to get a list of supported models and instructions
 Configure your `esp32` board for your robot.
 
 {{% /manualcard %}}
+{{% card link="/manage/configuration/" %}}
+{{% card link="/tutorials/" %}}
```
Comment (npentrel) [inline docs/installation/prepare/microcontrollers/_index.md:60]:
```suggestion
{{% card link="/tutorials/" %}}
{{% card link="/program/" %}}
```

File: docs/installation/prepare/rpi-setup.md [hunk 9]
```diff
@@ -208,6 +208,14 @@ If you are using hardware that requires these protocols, you must enable support
 
 {{< readfile "/static/include/install/install-linux.md" >}}
 
+## Next Steps
+
+{{< cards >}}
+{{% card link="/manage/configuration/" %}}
+{{% card link="/tutorials/" %}}
+{{% card link="/try-viam/" %}}
```
Comment (npentrel) [inline docs/installation/prepare/rpi-setup.md:216]:
```suggestion
{{% card link="/program/" %}}
```

**Pull request merged**